components/steps/Steps.tsx

```diff
@@ -1,17 +1,23 @@
-import Step1 from '@/components/steps/Step1'
-import Step2 from '@/components/steps/Step2'
-import Step3 from '@/components/steps/Step3'
-import Step4 from '@/components/steps/Step4'
+import { Step1Icon, Step2Icon, Step3Icon, Step4Icon, useStepIcon } from '@/components/steps/icons'
```

Comment:
Optional: I've grouped all icons together. It'd be even better to just name the icons by what they present, and just use them inside the `Steps` component.